app/models/user.rb

```diff
@@ -1,14 +1,15 @@
 class User < ActiveRecord::Base
-  validates_presence_of :first_name
-  validates_presence_of :last_name
+  validates :first_name, presence: true
+  validates :last_name, presence: true
   has_many :endorsements
   has_many :user_skills
   has_many :questions
   has_many :skills, through: :user_skills
+  has_many :conversations, foreign_key: :sender_id
 
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable and :omniauthable
-  devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :trackable, :validatable
-
+  devise :database_authenticatable, :registerable, :recoverable,
+         :rememberable,             :trackable,   :validatable,
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Unnecessary spacing detected.